src/algo/call_stack_keeper.cc

```diff
@@ -1,5 +1,6 @@
 #include "algo/call_stack_keeper.h"
 #include "err.h"
+#include <string>
```

Comment:
Please run `./tools/checkstyle` and fix include order (`<>` before `""`, includes are sorted lexicographically, relevant header is included before all else).